downloadDirectory/src/main/java/org/reactome/release/downloadDirectory/GenerateGOAnnotationFile/BiologicalProcessAnnotationBuilder.java

```diff
@@ -118,25 +117,25 @@ private static void processProteins(Set<GKInstance> proteinInstances, GKInstance
     /**
      * Before creating GOA line for BP annotations, the reaction in question needs to be checked for the existance of a 'goBiologicalProcess' attribute. If there is none
      * than the instance's 'hasEvent' referrals are checked for any.
-     * @param referenceEntityInst -- ReferenceEntity instance from protein instance.
-     * @param reactionInst
-     * @param taxonIdentifier -- CrossReference ID of protein's species.
-     * @throws Exception
+     * @param referenceEntityInst -- GKInstance, ReferenceEntity instance from protein instance.
+     * @param reactionInst -- GKInstance, parent reaction instance.
+     * @param taxonIdentifier -- String, CrossReference ID of protein's species.
```

Comment:
An example of a taxon identifier would be great so it's clear what kind of identifier is being used.